src/Tooltip/Tooltip.js

```diff
@@ -1,71 +1,75 @@
 /** @jsx jsx */
-import React from 'react'; /* eslint-disable-line no-unused-vars */
-import { Global, css, jsx } from '@emotion/core';
-import { ThemeProvider, withTheme } from 'emotion-theming';
+import { useState } from 'react'; /* eslint-disable-line no-unused-vars */
+import { jsx } from '@emotion/core';
 import PropTypes from 'prop-types';
-import 'react-tippy/dist/tippy.css';
-import { Tooltip as TippyTooltip } from 'react-tippy';
-
-import StyledTooltipContent from './StyledTooltipContent';
-
-const globalTippyStyles = (theme) => {
-  const arrowColor = theme.colors.monochrome.grey90;
-
-  return css`
-    .tippy-popper {
-      .tippy-tooltip.arbor-theme {
-        background: none;
-        padding: 0;
-      }
-    }
-
-    .tippy-popper[x-placement='bottom'] .tippy-tooltip.arbor-theme [x-arrow] {
-      border-bottom-color: ${arrowColor};
-    }
-
-    .tippy-popper[x-placement='top'] .tippy-tooltip.arbor-theme [x-arrow] {
-      border-top-color: ${arrowColor};
-    }
-
-    .tippy-popper[x-placement='left'] .tippy-tooltip.arbor-theme [x-arrow] {
-      border-left-color: ${arrowColor};
-    }
-
-    .tippy-popper[x-placement='right'] .tippy-tooltip.arbor-theme [x-arrow] {
-      border-right-color: ${arrowColor};
-    }
-  `;
-};
+import Tippy from '@tippyjs/react';
+import 'tippy.js/dist/tippy.css';
+import { animateFill, sticky } from 'tippy.js';
+import styled from '@emotion/styled';
 
-const Tooltip = ({ content, children, theme, ...props }) => {
-  const tooltipContent = (
-    <ThemeProvider theme={theme}>
-      <StyledTooltipContent>{content}</StyledTooltipContent>
-    </ThemeProvider>
-  );
+const StyledTippy = styled(Tippy)`
+  background: none;
+  padding: 0;
+
+  &[data-placement^='top'] > .tippy-arrow::before {
+    border-top-color: ${(props) => props.theme.colors.monochrome.grey90};
```

Comment:
We might want to turn this back into a variable in case we ever decided to change the color, but maybe that's unlikely enough that it doesn't matter.

Reply:
gonna leave this decision to @mayravherman 
could do it in another commit.